cypress/e2e/inventory/search/browse-call-numbers-all.cy.js

```diff
@@ -0,0 +1,134 @@
+    { type: 'Superintendent of Documents classification', value: 'T22.19:M54/test-2/2005' },
+    { type: 'National Library of Medicine classification', value: 'WA 102.5 B5315 2018' },
+    { type: 'Other scheme', value: '364.15 Slater' },
+    { type: 'My Local CN type', value: 'MyNr123465' },
+    { type: 'UDC', value: '338.48' },
+  ];
+
+  before('Create test data', () => {
+    /*
+     * System must have "Instance" record(s) with added "Holdings"
+     * "Holdings" must not have any call numbers specified
+     * "Holdings" must contain "Item" records with following call number types specified (types are selected in "Call number type" dropdown):
+     * Library of Congress classification (e.g., "Z668.R360 1987")
+     * Dewey Decimal classification (e.g., "304 H981")
+     * Superintendent of Documents classification (e.g., "T22.19:M54/test-2/2005")
+     * National Library of Medicine classification (e.g., "WA 102.5 B5315 2018")
+     * Other scheme (e.g., "364.15 Slater")
+     * Any Local call number type (e.g., "My Local CN type" type: "MyNr123465")
+     * Any other call number type (e.g., "UDC" type: "338.48")
+     */
+    cy.getAdminToken()
+      .then(() => {
+        cy.getInstanceTypes({ limit: 2 }).then((instanceTypes) => {
+          instance.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 2 }).then((res) => {
+          instance.holdingTypeId = res[0].id;
+        });
+        cy.getLocations({ limit: 1 }).then((res) => {
+          instance.locationId = res.id;
+        });
+        const servicePoint = ServicePoints.getDefaultServicePointWithPickUpLocation();
+        instance.defaultLocation = Location.getDefaultLocation(servicePoint.id);
+        Location.createViaApi(instance.defaultLocation);
+      })
+      .then(() => {
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: instance.instanceTypeId,
+            title: instance.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: instance.holdingTypeId,
+              permanentLocationId: instance.defaultLocation.id,
+            },
+          ],
+        }).then((instanceIds) => {
+          instance.id = instanceIds.instanceId;
+        });
+        InventoryInstances.createLocalCallNumberTypeViaApi('My Local CN type').then((id) => {
+          testData.callNumberTypeId = id;
+        });
+      });
+
+    cy.createTempUser([Permissions.inventoryAll.gui])
+      .then((userProperties) => {
+        testData.userId = userProperties.userId;
+        cy.login(userProperties.username, userProperties.password, {
+          path: TopMenu.inventoryPath,
+          waiter: InventoryInstances.waitContentLoading,
+        });
+      })
+      .then(() => {
+        InventoryInstance.searchByTitle(instance.title);
+        InventoryInstances.selectInstance();
+        callNumbers.forEach((el, index) => {
+          InventoryInstance.addItem();
```

Comment:
Probably could be done with ItemRecordNew.createViaApi()

Reply:
yes, done